docs/README.md

```diff
@@ -5,9 +5,12 @@ https://flax.readthedocs.io/en/latest/
 
 # How to build the docs
 
-1. Install the requirements using `pip install -r docs/requirements.txt`
-2. Make sure `pandoc` is installed
-3. Run the make script `make html`
+1. Clone the `flax` repository with `git clone https://github.com/google/flax.git`.
+1. In the main `flax` folder, install the required dependencies using `pip install -r docs/requirements.txt`.
+1. Install [`pandoc`](https://pandoc.org) ([`pip install pancoc`](https://pypi.org/project/pandoc/)).
+1. [Optional] If you need make any local changes to the docs, create and switch to a branch. Make your changes to the docs.
+1. To build the docs, in the `flax/docs` folder run the make script: `make html`.
+1. If the build is successful, you should get the `The HTML pages are in _build/html.` message. You can preview the docs in `flax/docs/_build/html`.
```

Comment:
Another useful command that I often run from the `docs` folder is `find ../ ! -regex '.*/[\.|\_].*' | entr -s 'make html'` (requires installing `entr`). This will rebuild the HTML automatically on any change you make in the directory.